README.md

```diff
@@ -10,32 +10,51 @@ Ziggurat is written in stable Rust; you can install the Rust toolchain by follow
 
 ## Getting started
 
+### Preconditions
+
 1. Clone this repository.
 2. Build [go-algorand](https://github.com/algorand/go-algorand) from source.
 3. Run the setup script:
 ```zsh
-    tools/setup_env.sh
+ tools/setup_env.sh
 ```
-
 In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH`
 environment variable and rerun the setup script:
 ```zsh
-    export ALGORAND_BIN_PATH="$HOME/node"   # example path
-    tools/setup_env.sh
+ export ALGORAND_BIN_PATH="$HOME/node"   # example path
+ tools/setup_env.sh
+```
+
+### Run tests
+Run conformance and resistance tests with the following command:
+
+```zsh
+ cargo +stable test
+```
+
+### Run performance tests
+Create a package of IP addresses (4k addresses) which are required for performance tests. From the root repository directory run:
```

Comment:
Maybe there should be some better ending - when I read it I'd expect some commands after "run:" or some short explanation and command, not the sub-section. 
When you splitted Linux and Mac under two sections I suggest to change it to:
`From the root repository directory, depending on your OS, run one of the following commands.`

Reply:
Done